website/docusaurus.config.js

```diff
@@ -146,6 +146,39 @@ const config = {
         darkTheme: darkCodeTheme,
         additionalLanguages: ['kotlin'],
       },
+      algolia: {
+        // The application ID provided by Algolia
+        appId: 'GBVAB0K8M8',
+
+        // Public API key: it is safe to commit it
+        apiKey: '4f04967a5a1f4b277d59d741a077affd',
+
+        indexName: 'oss-review-toolkit',
+
+        // Optional: see doc section below
+        contextualSearch: true,
+
+        // Optional: Specify domains where the navigation should occur through window.location instead on history.push.
+        // Useful when our Algolia config crawls multiple documentation sites and we want to navigate with
+        // window.location.href to them.
+        //externalUrlRegex: 'external\\.com|domain\\.com',
+
+        // Optional: Replace parts of the item URLs from Algolia. Useful when using the same search index for multiple
+        // deployments using a different baseUrl. You can use regexp or string in the `from` param. For example:
+        // localhost:3000 vs myCompany.com/docs
+        // replaceSearchResultPathname: {
+        //   from: '/docs/', // or as RegExp: /\/docs\//
+        //   to: '/',
+        // },
+
+        // Optional: Algolia search parameters
+        //searchParameters: {},
+
```

Comment:
The descriptions read as if we'd want to drop these for the time being...

Reply:
I thought it's useful to just copy the whole config template to make it easier if we need those options in future. But I can remove it if you prefer?

Reply:
I was unclear to me that this is the *whole* template. I though you manually kept *some* options that you felt could become useful in the future.

But in either case, I guess I'd prefer to drop them, also as this might go out of sync when new options are introduced upstream.

Reply:
I have also removed some redundant comments.